Packages/DesignSystem/Sources/DesignSystem/Font.swift

```diff
@@ -66,3 +73,14 @@ public extension Font {
     customFont(size: userScaledFontSize(baseSize: caption), relativeTo: .caption)
   }
 }
+
+
+
+extension UIFont {
```

Comment:
Here too

Reply:
No. That's weird. Let me sort it along with the merge conflict.